README.md

```diff
@@ -306,6 +306,18 @@ if (!detachResult.status) {
 }
 ```
 
+### Opening an existing .sqlite DB file
+
+By default in iOS the library looks intro [`NSDocumentDirectory`](https://github.com/ospfranco/react-native-quick-sqlite/blob/733e876d98896f5efc80f989ae38120f16533a66/ios/QuickSQLite.mm#L34-L35) and in Android it looks into `files` directory
```

Comment:
the `files` directory is part off the app bundle AFAIK, better to clarify this is not a system-wide files directory

Reply:
Looks is not the correct expresion, the passed database name is appended and generates a final URL: `${documentsDir}/${databaseName}`